### ISSUE TITLE
Some licenses are unchecked

### ISSUE THREAD

**Issue by mmurto** [doubleopen-project/policy-configuration#5]
Some of the licenses contain a `property:unchecked` tag:

https://github.com/doubleopen-project/policy-configuration/blob/be233ea91c70a0bb4a0ab46ae8004fd2602b9cec/license-classifications.yml#L137-L140

This means that the categorizations for those licenses is not fully completed yet. All the unchecked licenses should be checked, the categorization for them adjusted if necessary and the `property:unchecked` tag removed.

**Comment by sschuberth** [comment]
I agree that we should get rid of the `property:unchecked`. Its description seem to be confusing in saying

> This property marking means that the properties of the license have not been checked.

Why categorize it then in the first place? What's the difference to "unhandled" licenses that fail the `licenseClassifications.isCategorized(license)` check?

@willebra can we clean this up and either fully categorize `property:unchecked` licenses, or remove them?
